docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: '3'
+services:
+  client:
+    build:
+      context: .
+      dockerfile: Dockerfile.dev
+    volumes:
+      - "./:/rcruitme-front"
```

Comment:
mounts your docker container to the current working dir. you can update src files and get hot reloading without having to rebuild image.